python/TestHarness/testers/Tester.py

```diff
@@ -144,8 +174,10 @@ def checkRunnableBase(self, options, checks):
 
     # Are we running only tests in a specific group?
     if options.group <> 'ALL' and options.group not in self.specs['group']:
+      self.setStatus(reason, 'SKIP')
       return (False, reason)
```

Comment:
So do we want to return a tuple here still? I thought we were getting rid of all this.